.storybook/stories/tree/tree-node.stories.ts

```diff
@@ -45,6 +41,10 @@ export default {
     onKeyDown: { control: { disable: true }, table: { disable: true } },
   },
   args: {
+    // inputs
+    clrExpandable: true,
+    clrExpanded: false,
+    clrSelected: 'not selectable',
```

Comment:
Set `clrDisabled` to `false` as well.

```suggestion
    clrDisabled: false,
    clrExpandable: true,
    clrExpanded: false,
    clrSelected: 'not selectable',
```

Reply:
Fixed.